curator-x-async/src/main/java/org/apache/curator/x/async/modeled/details/ModeledCacheImpl.java

```diff
@@ -42,11 +46,12 @@
 
 class ModeledCacheImpl<T> implements TreeCacheListener, ModeledCache<T>
 {
-    private final TreeCache cache;
+    private final CuratorCacheBridge cache;
     private final Map<ZPath, Entry<T>> entries = new ConcurrentHashMap<>();
     private final ModelSerializer<T> serializer;
     private final StandardListenerManager<ModeledCacheListener<T>> listenerContainer = StandardListenerManager.standard();
     private final ZPath basePath;
+    private final EnsureContainers ensureContainers;
```

Comment:
This is more question for the previous review, but why `TreeCache` support creating the parent nodes of the path (as containers) and `CuratorCache` isn't?

Reply:
I think it was copying PathChildrenCache. With PersistentWatchers it's not necessary to have the ZNodes exist as the watcher is independent. So, it's not needed with CuratorCache.